Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -18,12 +18,23 @@
 
 #include "LibFuzzer/FuzzerDefs.h"
 
+#include "Firestore/core/src/firebase/firestore/remote/serializer.h"
+
+using firebase::firestore::remote::Serializer;
+
 namespace {
 
+// Fuzz-test the deserialization process in Firestore. The Serializer reads raw
+// bytes and converts them to a model object.
+void FuzzTestDeserialization(const uint8_t *data, size_t size) {
+  // Blank intentionally.
```

Comment:
Nit: I would read this comment as if this function is intended to always have an empty body. I presume it's actually not the case, so perhaps rephrase to something like `// TODO(minafarid): implement`.

Reply:
Changed. Thanks